### PR TITLE
fix issue 177

### PR DESCRIPTION
**Pull request by yuhuichina**
fix issue 177, config pure_enums in blade.conf when generating cpp files
see here https://github.com/chen3feng/typhoon-blade/issues/177

---
This change is [Reviewable](https://reviewable.io/reviews/chen3feng/blade-build/474)

### CODE REVIEW

File: blade.conf [hunk 1]
```diff
@@ -104,6 +104,8 @@ thrift_library_config(
     thrift_libs=['#thrift', '#pthread'],
     # The extra include paths to be passed to thrift compiler.
     thrift_incs=[],
+    # https://github.com/chen3feng/typhoon-blade/issues/177
```
Comment (chen3feng) [inline blade.conf]: 这个注释建议不要加

File: src/blade/scons_helper.py [hunk 2]
```diff
@@ -1186,10 +1186,10 @@ def setup_thrift_builders(top_env, build_dir, thrift_bin, thrift_incs_str):
     compile_thrift_python_message = console.erasable('%sCompiling %s$SOURCE%s to python source%s' % \
         (color('cyan'), color('purple'), color('cyan'), color('end')))
 
-    thrift_bld = SCons.Builder.Builder(action=MakeAction(
-        '%s --gen cpp:include_prefix,pure_enums -I . %s -I `dirname $SOURCE`'
+    thrift_bld = SCons.Builder.Builder(action = MakeAction(
```
Comment (chen3feng) [inline src/blade/scons_helper.py]: =两侧不留空格是python传递命名参数的约定风格

**Comment by yuhuichina** [comment]
all requested changes are finished.

**Comment by chen3feng** [comment]
very good, thanks!

**Pull request merged**